elastic/kibana
# [Security Solution]Expandable flyout - Replace rule sections with new components

This PR updates rule preview panel in the document expandable flyout:

- Replaced rule details sections with the simplified components from https://github.com/elastic/kibana/pull/166158
- Added `itemRenderer` to allow custom render of the description list
- Removed `isPanelView` props from the rule detail read only components. It was added to accommodate the preview styling (https://github.com/elastic/kibana/pull/163027) 

**No UI change from this PR**

**How to test**
- Go to alerts page and generate some alerts
- Expand a row in the table, a flyout should appear
- Click `Show rule summary` to expand the rule preview panel
